src/content/api/accounts.mdx

```diff
@@ -2,8 +2,8 @@
 title: Accounts
 description: Account model and related endpoints
 nav:
-  path: API
-  order: 7
+  path: API/Accounts/Sub Accounts
```

Comment:
@AndyClifford how do you actually click on to the 'Accounts' page?
The click opens the disclosure so I think you need the page again under this OR the click needs to take you to the accounts page. Not sure if this is a typo

Reply:
This is a typo left in from some testing